### PR TITLE
Feat/6

### PR DESCRIPTION
**Pull request by marianzburlea**

### CODE REVIEW

**Review by MariaTeo — APPROVED** [review]
this looks fun!

File: power/_template/react-component/name.jsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import * as T from './name.type'
 import * as S from './name.style'
 
-const CamelCase = ({ children }) => {
+export const CamelCase = ({ children }) => {
```
Comment (marianzburlea) [inline power/_template/react-component/name.jsx]: ma-taaaaa

Comment (marianzburlea) [reply power/_template/react-component/name.jsx]: gata Marian, gata

**Review by iuliamalciu — APPROVED** [review]
👍

**Review by iuliamalciu — APPROVED** [review]
He he 🔢

**Review by marianzburlea — COMMENTED** [review]
bravo 💯

**Review by flivanov92 — APPROVED** [review]
💯

**Review by MariaTeo — APPROVED** [review]
Great

**Review by iuliamalciu — APPROVED** [review]
nice 🥇

**Pull request merged**